.mypy.ini

```diff
@@ -0,0 +1,32 @@
+[mypy]
+files = aiohttp_sse, tests
```

Comment:
We'll want to include examples as well (now or in a later PR).

```suggestion
files = aiohttp_sse, tests  # , examples
```

Reply:
added